src/cunumeric/sort/sort.cu

```diff
@@ -221,7 +235,11 @@ void thrust_local_sort(const VAL* values_in,
 
   if (indices_out == nullptr) {
     if (volume == sort_dim_size) {
-      thrust::sort(exec_policy, values_out, values_out + volume);
+      if (stable) {
```

Comment:
Just curious, does the code ever pass `false` to `stable`? I thought we decided to use `stable_sort` all the time, as that's seemingly faster than the non stable one. Note that the compile time is one of the reasons we want to avoid any unnecessary template expansions.

Reply:
`stable==false` is the default value for both `sort `and `argsort `(as quicksort is default kind in numpy) . `stable_sort ` was faster before we added the custom allocator. With the faster allocation both approaches have similar performance. 

I think we can re-evaluate the removal of sort once all benchmarking is done. For now it is quite handy to be able to switch at runtime.

Reply:
sounds good. let's keep this branch until we collect enough evidences that we can completely ditch non-stable sort.